node/hyperbahn/egress-nodes.js

```diff
@@ -71,15 +71,7 @@ EgressNodes.prototype.exitsFor = function exitsFor(serviceName) {
     var exitNodes = Object.create(null);
     for (var i = 0; i < k; i++) {
         var shardKey = serviceName + '~' + i;
-
-        // TODO ringpop will return itself if it cannot find
```

Comment:
This cannot be removed. What we really want to do is detect whether or not we are "bootstrapping the ring" and if we are we want to send Declined error frames.

We really need to distinquish between "I just started and am bootstrapping" vs "I am isolated in a network partition"